dq_tests/config-doc-BiRNN.yml

```diff
@@ -0,0 +1,37 @@
+# Input data params
+TASK_NAME: 'testData-doc'                         # Task name
+SRC_LAN: 'src'                                     # Language of the source text
+TRG_LAN: 'mt'                                      # Language of the target text
+# SRC_LAN or TRG_LAN are expected to be the file extensions of the data files
+
+SEED: 1                                            # Value to set seed for deterministic results. (omit for random results)
+
+OUTPUTS_IDS_DATASET: ['doc_qe']
+OUTPUTS_IDS_MODEL: ['doc_qe']
+
+# Dataset class parameters
+SECOND_DIM_SIZE: 43
+PRED_SCORE: 'hter'                                  # used as file extension of the score files
+OUT_ACTIVATION: 'relu'                           #set as ‘relu’ function if predicted scores are in (0, +infinity), as a ‘sigmoid’ function for scores in (0,1) (for example, BLEU or HTER), or as a linear’ function for scores in (-infinity, +infinity).
+
+# Evaluation params
+METRICS: ['qe_metrics']                             # Metric used for evaluating the model
+EVAL_ON_SETS: ['val']                               # Possible values: 'train', 'val' and 'test' (external evaluator)
+NO_REF: False                                       # True if there is no reference data
+
+PRED_WEIGHTS: 'trained_models/testData-doc_srcmt_EncDoc/epoch_3.h5'
+
+MAX_INPUT_TEXT_LEN: 70                              # Maximum length of the input sequence
+
+MAX_OUTPUT_TEXT_LEN: 70                             # Maximum length of the output sequence
+                                                    # set to 0 if we want to use the whole answer as a single class
+
+PATIENCE: 5                                         # We'll stop if the val STOP_METRIC does not improve after this number of evaluations
+BATCH_SIZE: 5                                      # Size of each minibatch
+
+DOC_DECODER_HIDDEN_SIZE: 50
+
+STOP_METRIC: 'pearson'                              # Metric for the stop
+
+# Model parameters
+MODEL_TYPE: 'EncSent'                               # Model to train.
```

Comment:
Because of the way the preprocessing of docs is now done, doc training uses the EncSent model.
In old dQ, the EncDoc model just preprocessed the text then used the EncSent model anyway.
So either we can keep it this way, or define an EncDoc model which is just a straight inheritance of EncSent with no changes.
Though I suggest we don't do this in this PR.